test/cppsuite/configs/operations_test_stress.txt

```diff
@@ -1,14 +1,14 @@
 # Used as a stress test for the framework.
-duration_seconds=1500,
-cache_size_mb=1000,
+duration_seconds=600,
+cache_size_mb=2000,
 compression_enabled=true,
 metrics_monitor=
 (
     stat_db_size=
     (
         #At the end of the run the data files are approximately 2.3MB each. Which is a total of:
         #1.15GB, the history store isn't significant. Give the workload an extra 200MB of margin.
-        max=1350000000,
+        # max=1350000000,
         # FIXME-WT-8886 - This check has been disabled to remove noisy failures in evergreen and
         # will be properly corrected in WT-8886.
         runtime=false,
```

Comment:
Next line where `save` is set seems to have the wrong indentation

Reply:
Couldn't see what you were referring to here.

Reply:
Maybe it's GitHub that is showing a wrong indentation, that's fine.